### ISSUE TITLE
dev:console command outputs "must be compatible with" error

### ISSUE THREAD

**Issue by markshust** [netz98/n98-magerun2#901]
## Describe the bug

Running `n98-magerun2 dev:console` produces the following error:

```
n98-magerun2 dev:console

Fatal error: Declaration of N98\Magento\Command\Developer\Console\Shell::run(?Symfony\Component\Console\Input\InputInterface $input = NULL, ?Symfony\Component\Console\Output\OutputInterface $output = NULL) must be compatible with Psy\Shell::run(?Symfony\Component\Console\Input\InputInterface $input = NULL, ?Symfony\Component\Console\Output\OutputInterface $output = NULL): int in phar:///var/www/html/bin/n98-magerun2.phar/src/N98/Magento/Command/Developer/Console/Shell.php on line 41
```

## Expected behaviour

Drops into dev console without error.

## Steps to reproduce the issue

Install docker-magento project and run `bin/devconsole` (which is a shortcut for `bin/n98-magerun2`, which runs this script in the Docker container).

## Technical details

MacOS w/Docker Desktop, Magento 2.4.3-p1, PHP 7.4.26, n98-magerun2 4.9.0

**Comment by markshust** [comment]
See related PR. I'm not sure how to test these updates locally, but the PR should update the script so the function sig matches the parent function sig.

**Comment by cmuench** [comment]
@markshust Thanks for reporting. Seems that we need a automated test here. I will release a hotfix.

**Comment by cmuench** [comment]
@markshust Released as v4.9.1. We had to fight with the recent Composer 2.2 release first.

**Comment by cmuench** [comment]
@markshust for documentation. The automated test of this command is now in the functional test suite. See MR #922
